src/style.css

```diff
@@ -1,3 +1,96 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
 body {
   margin: 0;
+
+  --header-height: 60px;
+  --font-size: 12px;
+  --link-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  height: auto;
+  max-width: 1200px;
+  min-width: 1024px;
+  width: 100%;
+  padding: 4.4px 50px;
+  box-sizing: border-box;
+  align-items: center;
+  outline: none;
+}
+
+.header:hover {
+  outline: 3px solid #00acdc;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  outline: none;
+}
+
+.logo:hover {
+  outline: 3px solid #00acdc;
+}
+
+.nav {
+  display: flex;
+}
+
+.nav-list {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.list {
+  list-style-type: none;
+  margin: 0;
+}
+
+.list:not(.last-list) {
+  margin-right: 20px;
+}
+
+.link {
+  text-decoration: none;
+  color: black;
+  font-size: var(--font-size);
+  line-height: 1;
+  font-weight: 500;
+  text-transform: uppercase;
+
+  /* margin-top:calc((var(--header-height) - var(--font-size)) / 2);
+  margin-bottom:calc((var(--header-height) - var(--font-size)) / 2); */
+}
+
+.link:is(.link-blue) {
+  color: var(--link-color);
+}
```

Comment:
The `:is` pseudo-class is not necessary here and can be simplified. You can directly use `.link-blue` to apply the styles.